### PR TITLE
Modify access log, correcting response size element (was duplicated) …

### LINKED ISSUE #21: Log4j dependency
Currently this library has a dependency to log4j and I think It only needs to be slf4j.
The logger implementation should be whatever the user of this library needs/wants.

As a workaround excluding log4j seems to work.

### PR DESCRIPTION
**Pull request by ramonibz**
Modify access log, correcting response size element (was duplicated) and adding Zipkin trace information.

- Removed log4j library should fix #21 

@padilo @fiunchinho @aramirez-es @victuxbb Can you take a look please?

### CODE REVIEW

File: src/main/java/com/scmspain/karyon/accesslog/formatters/SchibstedApacheLog.java [hunk 1]
```diff
@@ -14,12 +16,20 @@
...
+
     return String.format(
-      "%s %d %d",
+      "%s %d \"%s\" \"%s\"",
       combinedApacheLogFormatter.format(logLine),
-      logLine.responseSize(),
-      logLine.timeTaken()
+      logLine.timeTaken(),
+      valueOrDefault(logLine.zipkinParentId()),
+      valueOrDefault(logLine.zipkinTraceId())
```
Comment (aramirez-es) [inline src/main/java/com/scmspain/karyon/accesslog/formatters/SchibstedApacheLog.java]: @fiunchinho Is Logstash input ready to support this format?

**Comment by fiunchinho** [comment]
Logstash2 is ready for this format, but not Logstash1. The idea is to release a new major version of karyon2-accesslog, since it will break the contract

**Comment by padilo** [comment]
Then if this PR is accepted we should increase major version and warn that this new version is not backward compatible.

**Comment by aramirez-es** [comment]
Ok from my side.

**Pull request merged**